libs/wingc/src/type_check.rs

```diff
@@ -1279,6 +1291,7 @@ impl<'a> TypeChecker<'a> {
 				self.type_check_exp(e, env, stmt.idx);
 			}
 			StmtKind::Assignment { variable, value } => {
+				// TODO: check if variable can be assigned to
 				let exp_type = self.type_check_exp(value, env, stmt.idx);
 				let var_type = self.resolve_reference(variable, env, stmt.idx);
```

Comment:
I think this should be easy to do. `resolve_reference` can return the `kind` and we'll have it. If we don't do that change we just make sure we have an issue for it.

Reply:
#1186